.github/workflows/tests.yml

```diff
@@ -22,6 +22,10 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Install micromamba
+      env:
+        MAMBA_ROOT_PREFIX: ~/micromamba
+        MAMBA_EXE: $(pwd)/micromamba
```

Comment:
@wolfv pinging you just in case you haven't seen this before. It is curious that this works for  macOS but not Linux.